internal/data-ingest-cli/README.md

```diff
@@ -0,0 +1,47 @@
+# OTLP Data Ingest Tool
+
+## Overview
+
+The primary purpose of this CLI Tool is to assist in reproducing support cases and troubleshooting issues within the OpenTelemetry collector pipeline.
+It allows users to accurately replicate conditions under which issues occur by reading data in different formats and sending it to the collector.
+Additionally, the tool can receive OTLP data via an endpoint and store it in a JSON file for detailed inspection and analysis,
+making it easier to diagnose and resolve problems related to OTLP data handling.
+
+*Note:* This is a developer tool used mainly for debugging, and not intended for being used within a production environment.
+
+## Features
+
+- Read OTLP JSON data from a file: The tool can read OTLP data from a specified file containing data in different formats and send it to an OpenTelemetry collector.
+ Supported formats are:
+  - OTLP Json
+  - Systemd
```

Comment:
You probably mean statsd no ?
```suggestion
  - Statsd
```

Reply:
good catch, thanks, that is indeed what i meant